Checks/PWR006/README.md

```diff
@@ -56,6 +58,48 @@ void example() {
 }
 ```
 
+#### Fortran
+
+In the following code, arrays `A` and `B` are never written to. However, they
+are privatized and thus each thread will hold a copy of each array, effectively
+using more memory and taking more time to create private copies:
+
+```f90
+subroutine example()
+  implicit none
+  integer :: i
+  integer :: a(5) = [1, 2, 3, 4, 5]
+  integer :: b(5) = [6, 7, 8, 9, 10]
+  integer :: sum(5)
+
+  !$omp parallel do default(none) firstprivate(a, b) shared(sum)
+  do i = 1, 5
+    sum(i) = a(i) + b(i)
+  end do
+  !$omp end parallel do
+end subroutine example
+```
+
+To save memory, change their scope to shared. This may also prevent memory
+issues when using arrays, as codes may easily run out of memory for a high
+number of threads:
+
+```f90
+subroutine example()
+  implicit none
+  integer :: i
+  integer :: a(5) = [1, 2, 3, 4, 5]
+  integer :: b(5) = [6, 7, 8, 9, 10]
+  integer :: sum(5)
+
+  !$omp parallel do default(none) shared(a, b, sum)
```

Comment:
If you agree with my other comments, I'd add `private(i)`

Reply:
[Done](https://github.com/codee-com/open-catalog/compare/70d218ee94a154404de7f2047bac58ee0689635a..53da9a7b92e35919f1543c12f7650952d5c94324)!